### PR TITLE
fix hact trip calculation for current year

### PR DESCRIPTION
**Pull request by ntrncic**

### CODE REVIEW

**Comment by ntrncic** [comment]
solves #256

**Comment by robertavram** [comment]
looks good.. did you test?

File: EquiTrack/partners/models.py [hunk 1]
```diff
@@ -337,13 +337,11 @@ def cp_cycle_trip_links(self):
         crs = CountryProgramme.current()
         if self.partner_type == u'Government':
             return self.linkedgovernmentpartner_set.filter(
-                        trip__from_date__lt=crs.to_date,
-                        trip__from_date__gte=crs.from_date
+                        trip__from_date__year=datetime.datetime.now().year,
```
Comment (robertavram) [inline EquiTrack/partners/models.py]: store this (`datetime.datetime.now().year,`) in a var under line 337 `crs = ` and use it from there

**Comment by ntrncic** [comment]
@robertavram yes tested in my local.

**Pull request merged**